pkg/updatestatus/mco/controller_common_constants.go

```diff
@@ -0,0 +1,83 @@
+// Taken from: https://github.com/openshift/machine-config-operator/blob/11d5151a784c7d4be5255ea41acfbf5092eda592/pkg/controller/common/constants.go
+// TODO: Replace this file with the original MCO code when transitioning to server-side
```

Comment:
Do we still want to import the code from the mco repo?
We made some changes on some files in this folder but this file seems unchanged.

Reply:
I will handle this one later (still within this PR, I just want to address other comments first).

Reply:
Can be addressed in a followup, or not at all (I agree that #1140 is very heavy for the value it brings)